src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -73,24 +98,52 @@
 			</Modal>
 
 			<!-- out of the avatar__options because of the overflow hidden -->
-			<Actions :open="opened" class="contact-avatar-options__popovermenu">
-				<ActionButton v-if="!isReadOnly" icon="icon-upload" @click="selectFileInput">
+			<!-- attention, this menu exists twice in this file -->
+			<Actions
+				default-icon="icon-picture-force-white"
+				:open="opened"
```

Comment:
```suggestion
				:open.sync="opened"
```

Reply:
Awesome - that was exactly what was missing. :+1: 
Thank you!